src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -203,7 +204,7 @@ private function updateGetOperation(\ArrayObject $pathOperation, array $mimeType
                 ],
             ];
 
-            if (!isset($pathOperation['parameters']) && count($parameters = $this->getFiltersParameters($resourceClass, $operationName, $resourceMetadata)) > 0) {
+            if (!isset($pathOperation['parameters']) && count($parameters = $this->getFiltersParameters($resourceClass, $operationName, $resourceMetadata, $responseDefinitionKey)) > 0) {
```

Comment:
The call to count can be removed and the related test (`> 0`) (perf improvement).